lib/iris/tests/results/file_load/known_loaders.txt

```diff
@@ -1,16 +1,16 @@
  * NetCDF OPeNDAP (priority 6)
  * NAME III (priority 5)
- * NetCDF_v4 (priority 5)
- * UM Post Processing file (PP) (priority 5)
  * NetCDF (priority 5)
  * NetCDF 64 bit offset format (priority 5)
+ * NetCDF_v4 (priority 5)
+ * UM Post Processing file (PP) (priority 5)
```

Comment:
Mostly they are mutually exclusive, but sadly not in the case of the ABF/L formats vs. the other "priority 3" formats. For example, a FieldsFile named "foo.abf" would load OK using the old order, but would try to load as an ABF file using the new order.